### ISSUE TITLE
produce a summary table of all datasets when compiling them all

### ISSUE THREAD

**Issue by LinguList** [glottobank/lexibank-data#42]
I imagine, having a big table on the website, showing some summary information on the datasets would be interesting, like:

| dataset | phonetic transcription | cognate sets | alignments | proto-forms |
| --- | --- | --- | --- | --- | 
| ptuk | y | y | y | y |
| bdpa | y | y | y | - |

This would basically expand the "datasets" view in the app, and transcription reports could be used to check for the specific properties, as a search for the content of cognate.csv for the latter. For proto-forms, we might need to add something specifically (?).

**Comment by xrotwang** [comment]
I guess this would also be useful in the repository, right? So for example when running `lexibank readme` for all datasets, it could create a global catalog `datasets.md`.

**Comment by LinguList** [comment]
yes, I think this would be cool. Given that it is running very long for all datasets, however, it might be useful to restrict the computed information only to those datasets which have been processed by cldf command. In this way, one could also work on the data step-by-step, or only consider those datasets that have changed in a local commit.

**Comment by xrotwang** [comment]
Regarding proto-forms, I'm not sure. I would think of these as the forms of a proto-language. And as such, the only special handling necessary would be a flag `proto=True` on the language. This would be different, if proto-forms could be just artificial constructs, e.g. constructed for any two wordlists without positing a corresponding language has ever existed.

**Comment by LinguList** [comment]
I'd say that my interest is more "linguistic" in making clear that proto-forms are in the data, as it is a quality statement, if you have aligned proto-forms (which we rarely have in any dataset). A statement that says that people who created the proto-forms actually try to respond in the greatest detail to their theory. But I guess, a proto=True would suffice for me as well. I'd just want to include it in my hierarchy of data-annotation, also to be used for the presentation. BTW: proto-forms can also occur without alignments, and ABVD shows them, for example, so the computationally different treatment is less important than the linguistic meaning of having such datsets...

**Comment by xrotwang** [comment]
Ok, I see. So what we'd ideally would want - I guess - is a display of alignments in a cognate set ordered genealogically, as it is done in [CSD](http://csd.clld.org/parameters/1170), right? 

So here's my proposal:
- We add a column `Proto` to `languages.csv` to make it possible to mark languages in the dataset as proto-languages.
- Still, proto-languages are linked to glottolog sub-groups (if possible).
- The Glottolog classification is used to order the list of counterparts in a cognate set.
- When a dataset includes proto-languages, this will be highlighted in its `README`.

**Comment by LinguList** [comment]
sounds good! and yes, I was thinking of something like CSD (even linear
ordering makes looking alignments easier if it's based on some kind of
underlying tree).

**Comment by xrotwang** [comment]
I'd add **License** and maybe **Macroareas** to the summary table.

**Comment by LinguList** [comment]
seems like a good idea! I remember I still need to change my license for the dataset without licence (in the other cases I checked, and it is GPL on Zenodo, so nothing to change there).

**Comment by xrotwang** [comment]
Well, if you are the maintainer, you could re-license the data in lexibank.

**Comment by LinguList** [comment]
okay, then I can do that. Only in those cases, where people gave the data before to me, I guess, I should not switch to commercial, so taking the cc-non-commercial license. I'll look into this and adjust them.
